tf2onnx/rewriter/custom_rnn_rewriter.py

```diff
@@ -105,13 +105,19 @@ def rewrite(self, context):
 
             state_inputs_initial_values = []
             for state_input in scan_props.state_inputs_initial_values:
-                nodes = self._adapt_scan_sequence_input_or_output("input", state_input, False)
-                state_inputs_initial_values.append(nodes[-1].output[0])
+                if self.g.opset == 8:
+                    nodes = self._adapt_scan_sequence_input_or_output("input", state_input, False)
+                    state_inputs_initial_values.append(nodes[-1].output[0])
+                else:
```

Comment:
how about add a check somewhere, if opset < 8, skip rewrite.
maybe in need_rewrite or at the beginning of rewrite?

it will make this ```else``` more meaningful.
otherwise, we may end up with Scan in a opset 7 model

Reply:
[optional] maybe also print some message like "rnn found, but needs higher opset"?